mobile/src/main/java/com/ataulm/wutson/discover/ShowSummaryView.java

```diff
@@ -32,22 +32,23 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
 
     @Override
     protected void onFinishInflate() {
-        View.inflate(getContext(), R.layout.merge_discover_by_genre_item, this);
+        View.inflate(getContext(), R.layout.merge_discover_show_summary, this);
 
-        posterImageView = (ImageView) findViewById(R.id.discover_by_genre_image_poster);
-        nameTextView = (TextView) findViewById(R.id.discover_by_genre_text_name);
+        posterImageView = (ImageView) findViewById(R.id.discover_show_summary_image_poster);
+        titleTextView = (TextView) findViewById(R.id.discover_show_summary_text_title);
     }
 
-    public void display(final Show show) {
+    void setPoster(String uri) {
```

Comment:
unrelated to naming but... views are dumb as fudge. Still got useful things like `setPoster` and `setTitle` rather than `setText` but now the views don't know about the model.